src/governance_engine/gsri_scoring_engine.py

```
@@ -0,0 +1,45 @@
+        """
+        Calculates the G-SRI score.
+        G-SRI > 0.85 triggers mandatory Basel IV capital surcharges.
+        """
+        score = (
+            inputs.get("resonance_drift", 0) * self.weights["resonance_drift"] +
+            inputs.get("cdi_delta", 0) * self.weights["cdi_delta"] +
+            inputs.get("agent_collusion_prob", 0) * self.weights["agent_collusion_prob"]
+        )
+
+        status = "NOMINAL"
+        if score > 0.5: status = "ELEVATED"
```

Ruff (0.15.15)

[error] Multiple statements on one line (colon)

(E701)